packages/jest-docblock/src/index.ts

```diff
@@ -18,6 +18,7 @@ const ltrimNewlineRe = /^(\r?\n)+/;
 const multilineRe = /(?:^|\r?\n) *(@[^\r\n]*?) *\r?\n *(?![^@\r\n]*\/\/[^]*)([^@\r\n\s][^@\r\n]+?) *\r?\n/g;
 const propertyRe = /(?:^|\r?\n) *@(\S+) *([^\r\n]*)/g;
 const stringStartRe = /(\r?\n|^) *\* ?/g;
+const STRING_ARRAY: Array<string> = [];
```

Comment:
This in theory is more performant as it's not allocating a new array every time, but mainly it just lets us drop `as Array<string>`